src/ompl/tools/thunder/SPARSdb.h

```diff
@@ -433,6 +439,11 @@ namespace ompl
                 heuristicScaling_ = heuristicScaling;
             }
 
+            void setEdgeWeightsInRoadmap(const EdgeWeightsInRoadmap weights)
```

Comment:
nit: Naming; I'd expect this to actually set the weights not the mode of calculation.